app/views/layouts/_header.html.erb

```diff
@@ -68,6 +68,22 @@
 </div>
 
 <script type="text/javascript">
+document.addEventListener("DOMContentLoaded", function () {
+  var gettingStartedLink = document.querySelector("#navbar-dropdown-1, #navbar-dropdown-2");
+  gettingStartedLink.addEventListener("click", function () {
+    if (gettingStartedLink.classList.contains("rotate180")) {
+      gettingStartedLink.classList.remove("rotate180");
+    }
+    else
+    {
+        gettingStartedLink.classList.toggle("rotate180");
+    }
+  });
+ document.addEventListener("click", function () {
+    if (gettingStartedLink.classList.contains("rotate180")) {
+      gettingStartedLink.classList.remove("rotate180");
+    } });
+});
```

Comment:
1. There seems to be no element with `navbar-dropdown-2`
2. Give a meaningful id, maybe like `getting-started-dropdown` or anything better
3. properly format the code to match the code style

Reply:
sir changes done